tests/unit/amo/components/TestInstallButtonWrapper.js

```diff
@@ -83,92 +140,67 @@ describe(__filename, () => {
   };
 
   it(`calls getClientCompatibility with the add-on's current version if no version is supplied`, () => {
-    const addon = fakeAddon;
-
-    _loadVersions({ slug: addon.slug, versions: [addon.current_version] });
-
     const clientApp = CLIENT_APP_FIREFOX;
-    const _getClientCompatibility = jest.fn().mockReturnValue({
-      compatible: true,
-    });
 
     _dispatchClientMetadata({
       clientApp,
     });
 
-    render({
-      _getClientCompatibility,
-      addon: createInternalAddonWithLang(addon),
-    });
+    renderWithCurrentVersion();
 
     expect(_getClientCompatibility).toHaveBeenCalledWith({
-      addon: createInternalAddonWithLang(addon),
+      addon: createInternalAddonWithLang(fakeAddon),
```

Comment:
This reinforces my belief that `renderWithCurrentVersion` should accept an `addon` (which could default to `createInternalAddonWithLang(fakeAddon)` if the test really doesn't care about the `addon`). That way you could still define `addon` in this test, pass it into `renderWithCurrentVersion`, and make assertions based on it. It ends up being the same thing in practice, but we do try to be explicit in tests where we are making assertions about something, rather than relying on the fact that `fakeAddon` is the default.